packages/documentation/src/partials/backend-variables.mdx

```diff
@@ -18,6 +18,7 @@ import { LinkOut } from '@interledger/docs-design-system'
 | `USE_TIGERBEETLE`               | `backend.use.tigerbeetle`                                                                                                                                             | `true`                                                      | When `true`, a TigerBeetle database is used for accounting. When `false`, a Postgres database is used.                      |
 | `WEBHOOK_URL`                   | `backend.serviceUrls.WEBHOOK_URL`                                                                                                                                     | _undefined_                                                 | Your endpoint that consumes webhook events.                                                                                 |
 | `AUTH_SERVICE_API_URL`          | `backend.serviceUrls.AUTH_SERVICE_API_URL`                                                                                                                            | _undefined_                                                 | The service-to-service api endpoint on your Open Payments authorization server.                                             |
+| `OPERATOR_TENANT_ID  `          | _undefined_                                                                                                                                                           | _undefined_                                                 | The unique identifier of the operator. The operator has elevated permissions and can manage resources for other tenants.    |     |
```

Comment:
This table isn't rendering properly on the preview: https://deploy-preview-3639--brilliant-pasca-3e80ec.netlify.app/integration/deployment/services/backend-service/#required